pkg/spdx/package.go

```diff
@@ -189,6 +222,7 @@ func (p *Package) Render() (docFragment string, err error) {
 	}
 
 	// If files were analyzed, calculate the verification
+	filesTagList := map[string]*struct{}{}
```

Comment:
Can we make this a `[]string`?

Reply:
Done. One question: I use that map of empty structs a lot as a temp dict to look up things that have to be listed once instead of looping a slice over and over.

Is it more efficient to loop the slice?

Reply:
> Is it more efficient to loop the slice?

In theory yes, but practically I don't know.